client/grpc/tmservice/service_test.go

```diff
@@ -152,7 +152,7 @@ func (s IntegrationTestSuite) TestQueryBlockResultsByHeight() {
 	s.Require().Equal(0, len(txResult))
 
 	beginBlock := blockResultsRes.GetResBeginBlock()
-	s.Require().Equal(11, len(beginBlock.Events)) // coinbase event (6) + transfer mintModule to feeCollectorName(5)
+	s.Require().Equal(7, len(beginBlock.Events)) // coinbase event (6) + transfer mintModule to feeCollectorName(5) - foundation abci (4)
```

Comment:
Why are the foundation events being removed in this case?

Reply:
Because, by default, foundation tax would be zero, so the corresponding 4 events would be gone.